### PR TITLE
Texting improvements

### LINKED ISSUE #123: [UI Improvement] Two quick improvements: Kext not running dialog / Reference to BSD in About
**Is your feature request related to a problem? Please describe.**
Not really.

**Describe the solution you'd like**
a) A little user-friendlier message that itlmw.kext is not running.
b) Improvement to BDS Clause 3 by including a URL reference.

**Describe alternatives you've considered**
Can't think of any alternatives except ignore them.

**Additional context**
a) Please expand the text when the driver is not running to something more friendlier, for example:
"The necessary WLAN driver is not found active on your system. Please verify that itlmw.kext is correctly installed and running."

b) Please fix, for the sake of being covered by BSD License, the About window RTF file with a URL reference, too (either a single mention or copy-paste the _actual_ license):

> **License**
> 
> This software is distributed through the 3-Clause BSD License, otherwise known as New BSD License. Read more at https://opensource.org/licenses/BSD-3-Clause

or the other, fuller approach:

> **License**
> 
> Copyright (c)2020. All rights reserved.
> 
> This software is distributed through the 3-Clause BSD License, otherwise known as New BSD License. Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
> 
> 1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
> 
> 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
> 
> 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
> 
> THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Kext Not Found
BDS-3-Clause
Thank you!

### PR DESCRIPTION
**Pull request by igorkulman**
Fixes #123

### CODE REVIEW

File: HeliPort/Credits.rtf [hunk 1]
```diff
@@ -2,13 +2,13 @@
 \cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fswiss\fcharset0 Helvetica-Bold;\f2\fnil\fcharset0 HelveticaNeue;
 }
 {\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red0\green0\blue0;\red0\green0\blue233;
-}
+\red0\green0\blue0;}
 {\*\expandedcolortbl;;\cssrgb\c0\c0\c0\cname textColor;\csgray\c0\c0;\cssrgb\c0\c0\c93333;
-}
+\cssrgb\c0\c0\c0;}
```
Comment (Goshin) [inline HeliPort/Credits.rtf:7]: better to use the semantic color `textColor`.

Comment (igorkulman) [reply HeliPort/Credits.rtf]: Changed, not sure if correct though, first time editing a RTF "manually"

File: HeliPort/Credits.rtf [hunk 2]
```diff
@@ -66,6 +66,8 @@ Communication with\'a0{\field{\*\fldinst{HYPERLINK "https://github.com/zxystd/it
 
 \f1\b \cf2 License\
 \
+\pard\pardeftab720\partightenfactor0
 
-\f0\b0 BSD-3-Clause\
-}
\ No newline at end of file
+\f0\b0 \cf5 \cb1 This software is distributed through the 3-Clause BSD License, otherwise known as New BSD License. \
```
Comment (Goshin) [inline HeliPort/Credits.rtf:71]: you can simply remove `\cf5` and `\cb1` here, which means foreground & background color selection, then it will inherit the color set above.

File: HeliPort/Credits.rtf [hunk 3]
```diff
@@ -2,13 +2,13 @@
 \cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fswiss\fcharset0 Helvetica-Bold;\f2\fnil\fcharset0 HelveticaNeue;
 }
 {\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red0\green0\blue0;\red0\green0\blue233;
-}
+\red0\green0\blue0;}
 {\*\expandedcolortbl;;\cssrgb\c0\c0\c0\cname textColor;\csgray\c0\c0;\cssrgb\c0\c0\c93333;
-}
+\cssrgb\c0\c0\c0\cname textColor;}
```
Comment (Goshin) [inline HeliPort/Credits.rtf:7]: `textColor` is already defined, so you can safely revert the changes from L5-12.

**Comment by Goshin** [comment]
thanks
